app/Http/Middleware/RequestLogger.php

```diff
@@ -3,10 +3,17 @@
 namespace App\Http\Middleware;
 
 use Closure;
+use Psr\Log\LoggerInterface;
 
 class RequestLogger
 {
-    const LOG_FILEPATH = "storage/logs/requests.log";
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
```

Comment:
Inject in the logger interface and assign it to a private property. This is a standard dependency injection pattern.